### ISSUE TITLE
Add email or discord notifications on test failures

### ISSUE THREAD

**Issue by rfay** [ddev/github-action-add-on-test#29]
GitHub is mysterious about who gets notified on automated test failures.

We should consider adding capability to notify in discord when tests fail. It may be annoying if everybody inherits it.

However, ddev-redis-7 was failing all its HEAD tests all the way up to v1.23.0 and we maintainers had no idea. The fixes were easy.

**Comment by rfay** [comment]
I see a number of discord actions that might work, https://github.com/marketplace?category=&type=actions&verification=&query=sort%3Apopularity-desc+discord

**Comment by julienloizelet** [comment]
Regarding a Discord notification, it seems that some action require a discord webhook url (like [this one ](https://github.com/sarisia/actions-status-discord)) or a webhook token/id (like [this one](https://github.com/marketplace/actions/discord-message-notify) ).

I don't know how these webhooks work, but I'm guessing we don't want to hard-code these values, and I'm not sure we want to share them so the end user can save them in their repo secrets.

What do you think ?

For email notification, [this action](https://github.com/marketplace/actions/cinotify-email) looks good to me.

**Comment by rfay** [comment]
You're right that it might not work because of the required secret (webhook).

**Comment by rfay** [comment]
I worked a couple hours with ChatGPT to create https://github.com/ddev/ddev-add-on-monitoring and https://github.com/ddev/ddev-add-on-monitoring/blob/main/check-addons.sh

Now we have check-addons.sh set up in our icinga monitoring instance, and will automatically know when tests are failing on official add-ons.

It can be run against everything when we want. I did that and opened issues on the several add-ons which didn't have working or running tests.
